pkg/controller/controlplane/valuesprovider.go

```diff
@@ -272,21 +273,26 @@ func (vp *valuesProvider) GetStorageClassesChartValues(
 		}
 	}
 	values := make(map[string]interface{})
-	if providerConfig.StorageClasses != nil && len(providerConfig.StorageClasses) != 0 {
-		allSc := make([]map[string]interface{}, len(providerConfig.StorageClasses))
```

Comment:
As mentioned in the `StoregeClasses` helm chart: please keep the single SC list and populated it accordingly.

Reply:
Done